lib/ansible/cli/__init__.py

```diff
@@ -339,29 +340,7 @@ def ask_passwords(self):
         return (sshpass, becomepass)
 
     def normalize_become_options(self):
-        ''' this keeps backwards compatibility with sudo/su self.options '''
-        self.options.become_ask_pass = self.options.become_ask_pass or self.options.ask_sudo_pass or self.options.ask_su_pass or C.DEFAULT_BECOME_ASK_PASS
-        self.options.become_user = self.options.become_user or self.options.sudo_user or self.options.su_user or C.DEFAULT_BECOME_USER
-
-        def _dep(which):
-            display.deprecated('The %s command line option has been deprecated in favor of the "become" command line arguments' % which, '2.9')
-
-        if self.options.become:
-            pass
-        elif self.options.sudo:
-            self.options.become = True
-            self.options.become_method = 'sudo'
-            _dep('sudo')
-        elif self.options.su:
-            self.options.become = True
-            self.options.become_method = 'su'
-            _dep('su')
-
-        # other deprecations:
-        if self.options.ask_sudo_pass or self.options.sudo_user:
-            _dep('sudo')
-        if self.options.ask_su_pass or self.options.su_user:
-            _dep('su')
+        pass
```

Comment:
This isn't part of our stable API so we should remove the calls to it and remove the method itself.

Reply:
i'm fine with removing it, i was just trying to be 'nice'